pkg/controllers/managementuser/settings/controller.go

```diff
@@ -42,13 +46,18 @@ func (h *handler) onChange(key string, obj *v3.Setting) (runtime.Object, error)
 		return nil, nil
 	}
 
-	return obj, h.apply.
+	err := h.apply.
 		WithOwner(obj).
+		WithCacheTypes().
```

Comment:
It requires a cache to be passed in

Reply:
https://github.com/rancher/rancher/blob/release/v2.6/pkg/controllers/management/authprovisioningv2/authprovisioningv2.go#L68

Reply:
Forgot to remove this. I did this above in Register() where I have access to the cache.